.vscode/settings.json

```diff
@@ -8,5 +8,8 @@
     "source.fixAll": true
   },
   "files.eol": "\n",
-  "files.insertFinalNewline": true
+  "files.insertFinalNewline": true,
+  "[javascript]": {
+    "editor.defaultFormatter": "vscode.typescript-language-features"
+  }
```

Comment:
What problem(s) do these lines solve?

Reply:
it just chooses the default formatter for the application.

You don't have to merge these if you don't want.